### DISCUSSION TITLE
how to read vital signs information on MIMIC III waveformDB using WFDB-python (or any other method)

### DISCUSSION THREAD

**Discussion by Michael-Fleicher-Etrog** [MIT-LCP/mimic-code#1374] [MIMIC-General]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Checked the online documentation: https://mimic.mit.edu/
    * Checked that your issue isn't already addressed: https://github.com/MIT-LCP/mimic-code/issues?utf8=%E2%9C%93&q=

### Description
Hey, I want to validate an algorithm on HR calculation using ECG signals from MIMIC III waveform and compare them to the HR related to those signals.
i know that MIMIC III waveform DB has also vital signs with a sample rate of 1/min, but i cant reach them.
1. what am I doing wrong? 
2. how can I reach the vital signs of each waveform using the WFDB-python library?
3. maybe I am wrong, and MIMIC III waveform DB has no vital signs information?

**Comment by briangow** [comment]
@Michael-Fleicher-Etrog , it isn't clear what you tried but here is an example of how to read and plot the heart rate numerics from the MIMIC-III waveform database for a particular subject in WFDB-python:

```
import wfdb
rec = wfdb.rdrecord('3000003n', pn_dir='mimic3wdb/30/3000003', channels=[0])
wfdb.plot_wfdb(record=rec)
```

Please see [wfdb.io](https://wfdb.io/) for more information on the WFDB package.
